### PR TITLE
Integration Domains: add tests to make sure Domains API works

### PR DESCRIPTION
**Pull request by f-trivino**
This commit implements unit tests to make sure the app Domain model, adapter, and view works well. Initially using factory-boy and Faker libraries for creating the test data.

Coverage: test domain __str__() method and adapter class so that the values for each of the fields in the serializer match the values for each of the fields on the model. Views unit tests are also added to make sure the domains API endpoints (GET, POST, PUT, and PATCH) work as expected.

### CODE REVIEW

**Comment by f-trivino** [comment]
@flo-renaud this one is ready to be reviewed.

File: .github/workflows/gating.yaml [hunk 1]
```diff
@@ -81,6 +75,23 @@ jobs:
...
         run: echo "Image pushed to ${{ steps.push-to-repo.outputs.registry-paths }}"
 
+      - name: Deploy the service
+        run: |
+          sudo -E su
+          podman run --name=bridge -d \
+                  --privileged \
+                  -p 8000:8000 -p 3501:3500 -p 4701:81 -p 4430:443 \
+                  --hostname bridge.ipa.test \
+                  --name bridge \
```
Comment (antoniotorresm) [inline .github/workflows/gating.yaml:85]: `--name` is duplicated

File: Dockerfile.test [hunk 2]
```diff
@@ -1,10 +1,12 @@
+# Podmanfile for building bridge image in development mode, including Swagger.
```
Comment (antoniotorresm) [inline Dockerfile.test:1]: What about renaming the file to `Containerfile.test`?

Comment (f-trivino) [reply Dockerfile.test:1]: I agree, done.

File: .github/workflows/gating.yaml (hunk 1 above)
Comment (f-trivino) [reply .github/workflows/gating.yaml:85]: good catch, fixed.

**Review by antoniotorresm — APPROVED** [review]
LGTM

**Pull request merged**